src/scripts/syncBreaches.js

```diff
@@ -72,10 +92,14 @@ export async function getBreachIcons(breaches) {
       console.log(`fetching: , ${logoFilename}`)
       const res = await fetch(
         `https://icons.duckduckgo.com/ip3/${breachDomain}.ico`);
-      await uploadToS3(logoFilename, Buffer.from(await res.arrayBuffer()))
-      const fileStream = createWriteStream(logoPath, { flags: 'wx' });
-      const bodyReadable = Readable.fromWeb(res.body)
-      await finished(bodyReadable.pipe(fileStream));
+      if (res.status === 200) {
+        await uploadToS3(logoFilename, Buffer.from(await res.arrayBuffer()))
+        const fileStream = createWriteStream(logoPath, { flags: 'wx' });
+        const bodyReadable = Readable.fromWeb(res.body)
+        await finished(bodyReadable.pipe(fileStream));
+      } else {
+        await deleteFromS3(logoFilename);
```

Comment:
Is there a reason we'd want to explicitly delete an old `Ico` if it exists? I think it would probably be safer keeping it in case `https://icons.duckduckgo.com/ip3` is temporarily offline (then our bucket will be emptied)

Alternatively, I'd suggest fallback on a google URI as a mechanism to fetch in case `icons.duckduckgo.com` is offline

Reply:
Yeah I'm not sure, but I at least wanted to delete the 404 icons that we've already uploaded to the dev bucket. But removed the deletion in de63bc95e.

Reply:
Thanks. Let me test this locally real quick